vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/TouchSteps.java

```diff
@@ -94,6 +94,17 @@ public void tapByLocator(Locator locator)
         findElementToTap(locator).ifPresent(touchActions::tap);
     }
 
+
+    /**
+     * Performs double tap on <b>element</b> located by <b>locator</b>
+     * @param locator locator to find an element
+     */
+    @When("I double tap on element located by `$locator`")
+    public void doubleTapByLocator(Locator locator)
+    {
+        findElementToTap(locator).ifPresent(touchActions::doubleTap);
```

Comment:
the validation message in `findElementToTap` may confuse users, it refers to simple `tap`